### ISSUE TITLE
[Bug Report] v-autocomplete pagination bug

### ISSUE THREAD

**Issue by havardthom** [vuetifyjs/vuetify#6691]
### Versions and Environment
**Vuetify:** 1.5.5
**Vue:** 2.6.8
**Browsers:** Chrome 71.0.3578.98
**OS:** Linux x86_64

### Steps to reproduce
In codepen: If you choose a state before starting to scroll the menu, the pagination will stop working and you cant scroll longer than 20 items. 
The bug is related to v-if="selectedState" prop in v-tooltip slot. If that v-if condition is removed, pagination works fine or if its changed to just true or false it works fine. So it is also related to using a variable from vuex specifically.

### Expected Behavior
v-autocomplete pagination should work even with a v-tooltip prepend slot using v-if condition having a vuex state variable.

### Actual Behavior
v-autocomplete pagination does not work with a v-tooltip prepend slot using v-if condition having a vuex state variable.

### Reproduction Link
<a href="https://codepen.io/anon/pen/bZqjMW?&editable=true&editors=101" target="_blank">https://codepen.io/anon/pen/bZqjMW?&editable=true&editors=101</a>

### Other comments
Comment from Kael in vuetify discord: "slot added -> re-renders menu for some reason -> no longer has scroll listener because it's added once on mount"

**Comment by KaelWD** [comment]
Duplicate of #11737
